docker-app/qfieldcloud/core/views/files_views.py

```diff
@@ -191,6 +192,9 @@ def post(self, request, projectid, filename, format=None):
                 changes={filename: [None, new_object.latest.e_tag]},
             )
 
+        # Delete the old versions if the files
```

Comment:
```suggestion
        # Delete the old file versions
```
I guess the `if` was a typo